### ISSUE TITLE
CLI: "Could not find compendium theme_config in plenti.json"

### ISSUE THREAD

**Issue by gedw99** [plentico/plenti#225]
load it
``` theme add https://github.com/plenti-themes/compendium ``` 

remove it:
```plenti theme remove compendium ``` 

always returns:
``` "Could not find compendium theme_config in plenti.json" ```

**Comment by jimafisk** [comment]
Ahh yes this could be related to some breaking changes I pushed recently (the themes sometimes fall behind and only work with older versions of Plenti). 

When I create a new site using that theme: `plenti new site delete-me --theme=https://github.com/plenti-themes/compendium`

And then try to `plenti serve` I get:
```
Error in DataSource build step
Can't create props for content/catgs/design.json 
Can't render htmlComponent for layouts_content_catgs_svelte
TypeError: Cannot read property 'socialLinks' of undefined
    at create_ssr:16:31
    at Object.$$render (create_ssr:1555:22)
    at create_ssr:61:118
    at $$render (create_ssr:1555:22)
    at Object.render (create_ssr:1563:26)
    at create_ssr:1:58
```

I just pushed an update (https://github.com/plenti-themes/compendium/commit/59754ff374945efbd640fc458ae102e5efbd3e55) to convert deprecated `_blueprint.json` files to `_defaults.json`.

Now if I retry `plenti new site delete-me --theme=https://github.com/plenti-themes/compendium` and `plenti serve` it seems to work.

Note: Updating the theme using `plenti theme update compendium` still threw the error. It's possible the theme isn't getting properly cleaned on update (https://github.com/plentico/plenti/issues/154).

**Comment by jimafisk** [comment]
Although on second look, your error seems like for some reason the `plenti.json` sitewide configuration file isn't getting updated. Make sure for the load command you're doing:

```
plenti theme add https://github.com/plenti-themes/compendium
```

(Note the addition of `plenti` to the front of the command).

That should add `theme` and `theme_config` keys to your `plenti.json`. Is that now working?

**Comment by gedw99** [comment]
He add updates the plentiful.json correctly 

the “remove” is the problem and cli displays an error, yet the plentiful.json does change 

Hope that helps.

**Comment by jimafisk** [comment]
Can you shared the contents of `plenti.json` after the `plenti theme add` command?

**Comment by gedw99** [comment]
Will do but tomorrow morning - is late here now

**Comment by jimafisk** [comment]
I just pushed a fix for this, it should be available in the next release (v0.5.17). Basically it was just an error message that was firing every time the `plenti theme remove` command was run, regardless of whether it failed or not. The command should actually be working in the current version despite the error message. Thanks for flagging this to me @gedw99!
